vscode-extension/media/welcomePage.md

```diff
@@ -1,4 +1,4 @@
-# Welcome to AIConfig Editor!
+# Tutorial!
```

Comment:
nit, "Welcome to AIConfig Editor" seems a better title since this is shown from welcome page command

Reply:
also, I guess do we need this page if we have the walkthrough?